src/player.py

```diff
@@ -1,2 +1,6 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
```

Comment:
Hey David , You are off to a good start with the assignment , Please make sure to reach out to the help channel or any of the PMs + myself, even your classmates if you need any assistance . I can't wait to see what you do with the project